misc/profiles2/trekking.brf

```diff
@@ -170,6 +291,12 @@ assign costfactor
   else if ( highway=motorway|motorway_link ) then   10000
   else if ( highway=proposed|abandoned     ) then   10000
 
+  #
+  # put a penalty on roads under constructions
+  # See https://github.com/abrensch/brouter/issues/106 and https://github.com/abrensch/brouter/issues/62
+  #
+  else if ( highway=construction     ) then   100
```

Comment:
Try to avoid (but not completely) roads under construction.

Reply:
Hmmm, it seems that `construction=yes` is deprecated (see https://wiki.openstreetmap.org/wiki/Key:construction#Usage).

`construction=minor` is mutually exclusive with `highway=construction` apparently but could be a good addition indeed (with a small penalty). In my experience (France), roads which would qualify for `construction=minor` are likely to have no usable cycleway (even if the road has one) so I guess the cost factor could be the one of the `highway` without `isbike` for these (if this local experience can be extended worldwide).

This would require to add the `construction` tag in `lookups.dat` file though.